From: Pravin B Shelar <pshelar@nicira.com>
Date: Thu, 17 Jul 2014 15:14:15 -0700
Subject: introduce NETIF_F_GSO_ENCAP_ALL helper mask
Patch-mainline: v3.17-rc1
Git-commit: f6eec614d2252a99b861e288b6301599d2d58da4 (partial)
References: bsc#1001486

Add NETIF_F_GSO_ENCAP_ALL mask covering all encapsulation GSO flags.

Signed-off-by: Pravin B Shelar <pshelar@nicira.com>
Acked-by: Andy Zhou <azhou@nicira.com>
Signed-off-by: Michal Kubecek <mkubecek@suse.cz>

SLE12-SP1: only introduce the helper, do not pick the openvswitch
change the original commit was about.

[js] only add more to already existing

---
 include/linux/netdev_features.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/netdev_features.h
+++ b/include/linux/netdev_features.h
@@ -164,6 +164,8 @@ enum {
 #define NETIF_F_SOFT_FEATURES	(NETIF_F_GSO | NETIF_F_GRO)
 
 #define NETIF_F_GSO_ENCAP_ALL	(NETIF_F_GSO_GRE |			\
+				 NETIF_F_GSO_IPIP |			\
+				 NETIF_F_GSO_SIT |			\
 				 NETIF_F_GSO_UDP_TUNNEL |		\
 				 NETIF_F_GSO_MPLS)
 
